docs/pages/getting-started/installation.mdx

```diff
@@ -0,0 +1,38 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Installation
+
+Crypta consists only of one binary without any dependencies.
+
+<Tabs
+  defaultValue="windows"
+  values={[
+    { label: 'Windows', value: 'windows', },
+    { label: 'Darwin', value: 'darwin', },
+    { label: 'Linux', value: 'linux', },
+  ]}
+>
+<TabItem value="windows">
+
+```powershell
+curl -L -o crypta.exe "https://github.com/codetent/crypta/releases/latest/download/crypta_Windows_x86_64.exe"
+```
+
+</TabItem>
+<TabItem value="darwin">
+
+```bash
+curl -L -o crypta "https://github.com/codetent/crypta/releases/latest/download/crypta_Darwin_x86_64" && sudo install -c -m 0755 crypta /usr/local/bin
+```
+
+</TabItem>
+<TabItem value="linux">
+
+```bash
+curl -L -o crypta "https://github.com/codetent/crypta/releases/latest/download/crypta_Linux_x86_64" && sudo install -c -m 0755 crypta /usr/local/bin
+```
```

Comment:
I think the uploaded artifacts currently only contain archives, and not the executables themselves. How does this work?

Reply:
I skipped archiving for now and uploaded the binaries directly. Additionally, I updated the documenation